apps/chainlit/app.py

```
@@ -2016,79 +2181,273 @@ async def set_chat_profiles():
+        ),
+        Tags(
+            id="active_keywords",
+            label="Schlüsselwörter",
+            description="Themen aus Ihrem Chatverlauf. Entfernen oder hinzufügen.",
+            initial=active_kw_values,
+        ),
+        Select(
+            id="regenerate_keywords",
+            label="Schlüsselwörter-Aktion",
+            description="Schlüsselwörter aus dem Chatverlauf neu extrahieren",
+            values=["– Keine Aktion –", "Jetzt neu generieren"],
```

Ruff (0.15.6)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)